main.cpp

```diff
@@ -28,510 +28,592 @@ Thing: Car Wash
+{
+    std::cout << "prescription needed" << std::endl;
+}
+
+struct Chord
+{
     int numberOfNotes = 4;
-//    2) possible next chords within key
     std::string nextChordList = "F#7, B";
-//    3) chord number in roman numerals
     std::string chordInRomanNumerals = "V7";
-//    4) has chord been altered
     bool alteredChord = false;
-//    5) possible preceding chords within key
     std::string precedingChordList = "C#m, D#m";
 
-//    1) make a sound
-    void makeSound(int NoteNumber);
-//    2) be played as arpeggio
+    int makeSound(int noteNumber);
     int playArpeggio(int numberOfHeldNotes);
-//    3) be omitted
-    void playNothing(std::string chord);
+    int playNothing(int numberOfNotes);
 };
 
-/*
-Thing 5) Keys
-5 properties:
-    1) key depressed
-    2) velocity of key pressed (1-127)
-    3) MIDI number of note pressed (1-127)
-    4) note legato value (note held, while another is played)
-    5) keyboard set to monophonic
-3 things it can do:
-    1) play legato
-    2) play chords
-    3) play monophonic
- */
+int Chord::makeSound(int noteNumber)
+{
+    if (noteNumber)
+    {
+        return noteNumber;
+    }
+    else
+    {
+        return playNothing(0);
+    }
+}
 
-struct Keys
+int Chord::playArpeggio(int numberOfHeldNotes)
+{
+    if(numberOfHeldNotes)
+    {
+        return numberOfHeldNotes;
+    }
+    else
+    {
+        return playNothing(0);
+    }
+}
+
+int Chord::playNothing(int numOfNotes = 0)
 {
+    return numOfNotes;
+}
 
-//    1) key depressed
+struct Keys
+{
     bool keyDepressed = false;
-//    2) velocity of key pressed (1-127)
     int noteVelocity = 67;
-//    3) MIDI number of note pressed (1-127)
     int noteMidiNumber = 59;
-//    4) note legato value (note held, while another is played)
-    float legatoValue = 1003.5;
-//    5) keyboard set to monophonic
+    float glideValue = 1003.5;
     bool monophonicSynth = true;
 
-//    1) play legato
-    void playLegato();
-//    2) play chords
-    void playChords();
-//    3) play monophonic
-    void playMonophonic(Keys keys);
+    bool playGlide(float glideValue);
+    void playChords(Keys keys);
+    bool playMonophonic(bool monophonicSynth);
 };
 
-/*
-Thing 6) Oscillator
-5 properties:
-    1) pitch value of oscillator
-    2) octave setting of oscillator (int 'foot' selection of LFO, 32, 16, 8, 4, 2)
-    3) oscillator volume level
-    4) oscillator pitch source
-    5) waveform of oscillator
-3 things it can do:
-    1) make a sound
-    2) be an LFO
-    3) be a square wave
- */
+bool Keys::playGlide(float glideNumber)
+{
+    if (glideNumber > 1.0f)
+    {
+        return playMonophonic(true);
+    }
+    return playMonophonic(false);
+}
+
+void Keys::playChords(Keys keys)
+{
+    keys.playMonophonic(false);
+}
+
+bool Keys::playMonophonic(bool monophonicKeys = true)
+{
+    if (monophonicKeys)
+    {
+        return monophonicSynth = true;
+    }
+    else
+    {
+    return monophonicKeys = false;
+    }
```

Comment:
See previous comments about why this code is awful
- [ ] revise

Reply:
Done.

Reply:
Done.